simulation/simulation_interface/src/conversions.cpp

```diff
@@ -232,15 +232,15 @@ void toMsg(const openscenario_msgs::ActionStatus & proto, openscenario_msgs::msg
 void toProto(
   const openscenario_msgs::msg::LaneletPose & pose, openscenario_msgs::LaneletPose & proto)
 {
-  proto.set_lanelet_id(pose.lanelet_id);
+  proto.set_lanlet_id(pose.lanelet_id);
   proto.set_s(pose.s);
   proto.set_offset(pose.offset);
   toProto(pose.rpy, *proto.mutable_rpy());
 }
 
 void toMsg(const openscenario_msgs::LaneletPose & proto, openscenario_msgs::msg::LaneletPose & pose)
 {
-  pose.lanelet_id = proto.lanelet_id();
+  pose.lanelet_id = proto.lanlet_id();
```

Comment:
lanlet

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/357/commits/1f8ce0894f6bc77d9cb558a17f03e44105d6f4dd